tests/subsys/net/lib/download_client/src/main.c

```diff
@@ -1,19 +1,24 @@
 /*
- * Copyright (c) 2022 Nordic Semiconductor ASA
- *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
- */
+* Copyright (c) 2022 grandcentrix GmbH
```

Comment:
And please don't replace copyrights. You can add your copyright just below the existing one.